taskcluster/script.py

```diff
@@ -151,6 +155,14 @@ def main():
     except subprocess.CalledProcessError as e:
         print('{} attempting netstat'.format(e))
 
+    # enable charging on device
+    #   see https://bugzilla.mozilla.org/show_bug.cgi?id=1565324
+    # TODO: detect the device type
```

Comment:
Some prior work that can be used to detect the model can be found here: https://github.com/mozilla/powerusage-android/blob/master/utils/adb_utils.py#L40-L48

Reply:
subprocess.call doesn't check the result code, but changed to mozdevice calls.